lib/reline/terminfo.rb

```diff
@@ -148,9 +136,14 @@ def self.tigetnum(capname)
     num
   end
 
+  # NOTE: This means Fiddle and curses are enabled.
   def self.enabled?
     true
   end
+
+  def self.supported?
```

Comment:
I think the name `supported?` is not clear.
`Terminfo.enabled?`: terminfo is enabled
`Terminfo.supported?`: ENV['TERM'] is supported in terminfo
How about `term_supported?`

Reply:
Sounds good. I updated the code as suggested.